examples/generative/corrdiff/generate.py

```diff
@@ -226,15 +234,18 @@ def generate_fn(image_lr):
                 with nvtx.annotate("regression_model", color="yellow"):
                     image_reg = generate(
                         net=net_reg,
-                        img_lr=image_lr_patch,
+                        img_lr=image_lr_patch[:, 0:16],
```

Comment:
Magic number. Should this be removed?

Reply:
That's the tricky part. Thanks for bringing it up. The pretrained regression model takes 12 channels + 4 positional embeddings as input. If a different positional embedding is utilized in the diffusion model, we'll need to retain both positional embeddings in the dataset and select them as required for regression and diffusion. Hard-coding isn't an elegant solution, but the diffusion training code lacks information regarding the positional embedding of the regression. Should we consider making it configurable?

Reply:
>  If a different positional embedding is utilized in the diffusion model, we'll need to retain both positional embeddings in the dataset and select them as required for regression and diffusion.

will this be fixed by moving the position encoding into the network rather than the dataset? See also https://github.com/NVIDIA/modulus/pull/401#discussion_r1537920232